app.py

```diff
@@ -93,24 +93,27 @@ def show_property_by_id(id):
     connection = get_flask_database_connection(app)
     repository = PropertyRepository(connection)
     property = repository.find_property_by_id(id)
-    return render_template('get_property.html', property=property)
+    
+    repository = BookingRepository(connection)
+    dates_taken = repository.dates_taken(id)
+    return render_template('get_property.html', property=property, dates_taken=dates_taken), 200
 
 @app.route('/properties/<int:id>', methods=['POST'])
 @login_required
 def create_booking(id):
     """
     If all details are valid, a property should be booked by filling in the booking form
     """
-    if not request.form.get('date_from') or not request.form.get('date_to'):
+    if not request.form.get('start') or not request.form.get('end'):
```

Comment:
just a suggestion for naming of arguments to be specific regarding the input . e.g start_date or end_date . Likely this will affect the html naming also